internal/pkg/plugin/trello/helper.go

```diff
@@ -0,0 +1,60 @@
+package trello
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+	
+	"github.com/merico-dev/stream/pkg/util/log"
+	"github.com/merico-dev/stream/pkg/util/trello"
+)
+
+func buildState(options *Options, ti *TrelloItemId) map[string]interface{} {
+	res := make(map[string]interface{})
+	output := make(map[string]interface{})
```

Comment:
The definition can be moved closer to the first usage (above line 20) so that it's easier to read.